templates/html.html.twig

```diff
@@ -48,13 +48,11 @@
     <css-placeholder token="{{ placeholder_token }}">
     <js-placeholder token="{{ placeholder_token }}">
   </head>
-  {% set classes = [] %}
+  {% set classes = [is_front ? 'front', not is_front ? 'not-front'] %}
```

Comment:
nice.

Reply:
Thanks @pookmish. Good catch, will keep this cleaner.